src/main.rs

```diff
@@ -297,13 +307,19 @@ fn main() -> anyhow::Result<()> {
             for project_path in &paths {
                 match remove_older_than(project_path, &keep_duration, dry_run) {
                     Ok(cleaned_amount) if dry_run => {
+                        total_cleaned += cleaned_amount;
                         info!("Would clean: {}", format_bytes(cleaned_amount))
                     }
-                    Ok(cleaned_amount) => info!("Cleaned {}", format_bytes(cleaned_amount)),
+                    Ok(cleaned_amount) => {
+                        total_cleaned += cleaned_amount;
+                        info!("Cleaned {}", format_bytes(cleaned_amount))
+                    }
                     Err(e) => error!("Failed to clean {:?}: {:?}", project_path, e),
                 };
             }
         }
+
+        info!("Total amount: {}", format_bytes(total_cleaned));
```

Comment:
If there's only one project, this will print the same thing as the `Cleaned` output above, right? Can we avoid printing it unless there's multiple projects?